plugins/config_manager.py

```diff
@@ -90,6 +90,11 @@ def configure(self):
                 self.opts.set_disabled or self.opts.add_repo):
             demands.root_user = True
 
+        # sanitize commas https://bugzilla.redhat.com/show_bug.cgi?id=1830530
+        temp_list = [x.split(',') for x in self.opts.crepo if x is not ',']
+        # flatten sublists
+        self.opts.crepo = [item for sublist in temp_list
+            for item in sublist if item is not '']
```

Comment:
Use rather "!=" instead of "is not", because you want to check if the strings are equal, not if the objects are the same. The result is the same here, but not in general.